### PR TITLE
cli: Print 'Error' when CLI command results in an error (#6)

### PR DESCRIPTION
**Pull request by jwhui**
This is a fix for #6.

> Print 'Error' when CLI command results in an error, rather than failing silently.

### CODE REVIEW

File: src/cli/cli.cpp [hunk 1]
```diff
@@ -136,6 +136,18 @@ int Interpreter::Hex2Bin(const char *aHex, uint8_t *aBin, uint16_t aBinLength)
...
 
+void Interpreter::AppendResult(ThreadError error)
+{
+    if (error == kThreadError_None)
+    {
+        sResponse.Append("Done\r\n");
+    }
+    else
+    {
+        sResponse.Append("Error\r\n");
```
Comment (darconeous) [inline src/cli/cli.cpp]: Is there any easy way to include the error code value in the printout for the error case?

**Comment by darconeous** [comment]
Would be nice if the error code could be printed out in the error case.

Otherwise, LGTM 👍

**Comment by jwhui** [comment]
@darconeous, updated to print out the error code as an integer.

**Comment by darconeous** [comment]
👍

**Pull request merged**